root/layout/components/sidebar/WorkSidebar.js

```diff
@@ -77,7 +77,7 @@ const WorkSidebar = ({$c, work}: Props) => {
               </SidebarProperty>
             ) : null}
 
-            {iswcs && iswcs.length ? (
+            {iswcs.length ? (
```

Comment:
So iswcs always exists, then? :)

Reply:
Looks like it to me. I see `iswcs => [map { $_->TO_JSON } $self->all_iswcs],` is output unconditionally in `Entity::Work::TO_JSON`.